.github/workflows/github-actions.yml

```diff
@@ -92,3 +93,9 @@ jobs:
         - name: Run tests
           run: |
             bundle exec rspec
+        - name: Upload screenshots
+          if: failure()
+          uses: actions/upload-artifact@v4
+          with:
+            name: capybara-screenshots
+            path: tmp/capybara
```

Comment:
Added to be able to see failure screenshots from system specs.

Reply:
Where will these screenshots be stored? Does this increase our billing?